src/Sylius/Bundle/AdminBundle/Form/Type/OrderType.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Form\Type;
+
+use Sylius\Bundle\OrderBundle\Form\Type\OrderType as BaseOrderType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+final class OrderType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('shippingAddress', AddressType::class, [
+                'add_build_address_form_subscriber' => $options['add_build_address_form_subscriber'],
```

Comment:
This parameter add_build_address_form_subscriber was added to prevent adding the subscriber in the admin panel. Now that we have separated the shop and admin, can this parameter be removed?

Reply:
Since it's still here was it forgotten or is something preventing us from removing it?